runtime/compiler/control/J9Options.cpp

```diff
@@ -898,6 +933,12 @@ TR::OptionTable OMR::Options::_feOptions[] = {
         TR::Options::setStaticNumeric, (intptr_t)&TR::Options::_iprofilerSamplesBeforeTurningOff, 0, "P%d", NOT_IN_SUBSET},
    {"itFileNamePrefix=",  "L<filename>\tprefix for itrace filename",
         TR::Options::setStringForPrivateBase, offsetof(TR_JitPrivateConfig,itraceFileNamePrefix), 0, "P%s"},
+#if defined(J9VM_OPT_JITSERVER)
+   {"jitserverLoadExclude=", "D{regex}\tdo not load methods beginning with xxx from the JITServer AOT cache",
```

Comment:
`jitserverAOTCacheLoadExclude`, `jitserverAOTCacheStoreExclude`. Also, does this actually only support "beginning with xxx", or wildcards as well?

Reply:
It should support exactly what the `exclude=` and `limit=` options support. I believe that includes wildcards.